docs/UserGuide.md

```diff
@@ -690,3 +679,15 @@ Furthermore, certain edits can cause the PROperty to behave in unexpected ways (
 - **CONTRACTOR**: Professionals providing construction, renovation, or repair services.
 
 [_Back to Top_](#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Appendix
+
+### Planned Enhancements
```

Comment:
I think we were planning to also add #315 (improved error handling for export) into future enhancements?

Reply:
I actually addressed #315 under the export command. Do you feel it belongs here instead?

Reply:
Discussed to add it into planned enhancements still on tele, pending #331 to be merged